SU-SWS/stanford_paragraph_types
# HSDO-934 Changed available pattern options for teaser cards.

# READY FOR REVIEW

# Summary
- Changed texture options on teaser cards

# Needed By (Date)
- 8/3

# Urgency
- Low

# Steps to Test

1. Checkout branch
2. database update & revert feature
3. create story page with teaser card paragraph type
4. in the "add some color" fieldset choose the new pattern options.
5. ensure they look ok.

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
